README.md

```diff
@@ -4,4 +4,11 @@
 
 * Feel free to take the code and create your own bot to host.
 * Uses a `.env` file to define the bot token. Syntax is `TOKEN=<discord bot token here>`
+    * This is currently broken. As a temporary solution, when testing you code locally, use the following procedure:
+    * In `bot.py`:
+        * Comment out the line `import token1 as token`
+        * At the bottom of the file, temporarily comment out the line `bot.run(token.stringToken())`
+        and replace it with `bot.run("<bot token here>")`. Remember to delete this temporary line afterwards,
```

Comment:
The concept is to have a `token.py` file with a function that returns the token. We'll be updating this soon